website/source/docs/cluster/bootstrapping.html.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: "docs"
+page_title: "Creating a cluster"
+sidebar_current: "docs-cluster-bootstrap"
+description: |-
+  Learn how to bootstrap a Nomad cluster.
+---
+
+# Creating a cluster
+
+Nomad clusters in production comprises of a few Nomad servers, clients and
+optionally Consul servers and clients.Before we start discussing the specifics
+around bootstrapping clusters we should discuss the network topology. Nomad
+models infrastructure as regions and datacenters. Regions may contain multiple
+datacenters. Servers are assigned to a single region and hence a region is a
+single scheduling domain in Nomad. Multiple regions can be connected together
+and users can target a different region when they query the Nomad APIs or submit
+a job to servers in a specific region. 
+
+
+## Nomad Servers
+
+Nomad servers are expected to have network latencies no more than 10
+milliseconds between them. Nomad servers could be spread across multiple
+datacenters, if they have low latency connections between them, to achieve high
+availability. For example, on AWS every region comprises of multiple zones which
+have very low latency links between them, so every zone can be modeled as a
+Nomad Data Center and every Zone can have a single Nomad server which could be
+connected to form a quorum and form a Region. Nomad servers uses raft for
+replicating state between them and raft being highly consistent needs a quorum
+of servers to function, therefore we recommend running an odd number of Nomad
+servers in a region. Usually running 3-5 servers in a region is recommended. The
+cluster can withstand a failure of one server in a cluster of three
+servers and two failures in a cluster of five servers. Adding more servers to
+the quorum adds more time to replicate state and hence throughput decreases so
+we don't recommend having more than seven servers in a region.
+
+During the bootstrapping phase Nomad servers need to know the addresses of other
+servers, which can be achieved by using the `-reconnect-join` cli command or by
+pointing Nomad servers to a Consul Service.
+
+
+## Nomad Clients
+
+Nomad clients are organized in DCs and they have to be seeded by the list of
+servers that they will have to connect.
+
+Operators can either place the addresses of the Nomad servers in the client
+configuration or point Nomad client to the Nomad server service in Consul. Once
+a client establishes connection with a Nomad servers, if new servers are added
+to the cluster the addresses are propagated down to the clients along with
+heartbeat.
+
+
+## Consul Cluster
+
+Bootstrapping a Nomad cluster becomes significantly easier if operators use
+Consul and registers Nomad servers with Consul. Network topology of a Consul
+cluster is slightly different than Nomad. Consul models infrastructures as Data
+Centers, and each Consul Data Center can have up to ~10,000 nodes and multiple
```

Comment:
To be consistent with the rest of our documentation so far: `Data Centers` -> `datacenters`.  I'd also remove the reference to 10K nodes/cluster since that is a moving target.  Those numbers were accurate for the 0.5 series, but aren't anymore.